codespell_lib/data/dictionary.txt

```diff
@@ -15506,6 +15510,8 @@ intenational->international
 intendet->intended
 inteneded->intended
 intension->intention
+intensional->intentional
+intensionally->intentionally
```

Comment:
Could also add the variants with a single `l` ?

Reply:
like abde9cf25a6c7af1b81c4eb027d17884bf0e2228?

Reply:
Yep :+1:   Could also add `intentialy` and `itentionaly`? :shrug: